src/components/accessories/patientTherapy/therapyForm/TherapyForm.tsx

```diff
@@ -1,4 +1,5 @@
 import { Checkbox, FormControlLabel } from "@material-ui/core";
+import id from "date-fns/esm/locale/id/index.js";
```

Comment:
Is this an import mistake or are you gonna use in somewhere?

Reply:
mistake